webpack/assets/javascripts/foreman_users.js

```diff
@@ -55,3 +55,12 @@ export function testMail(item, url, param = {}) {
     },
   });
 }
+
+export function authSourceSelected() {
+  const id = $('#user_auth_source_id option:selected').text();
```

Comment:
If you feel like refactoring a bit - if you pass `this` to the function on change as a parameter, `id` will be `param.selectedOptions[0].textContent` and then you could more easily test the function